tabby-ssh/src/components/keyboardInteractiveAuthPanel.component.scss

```diff
@@ -7,3 +7,18 @@
 .prompt-text {
     white-space: pre-wrap;
 }
+
+.totp-info {
```

Comment:
Please use a standard bootstrap `alert` component here (`.alert.alert-success`) so that it can respond to theme changes